Elsewhen/Views/MykeMode.swift

```diff
@@ -85,6 +86,35 @@ struct MykeMode: View, OrientationObserving {
         
     }
     
+    @ViewBuilder
+    var mykeModeButtons: some View {
+        Button(action: {
+            #if os(macOS)
+//                        WindowManager.shared.openSelectTimeZones(selectedTimeZone: .constant(TimeZone.current), selectedDate: $selectedDate, selectedTimeZones: $selectedTimeZones)
```

Comment:
What's this line? 🤔

Reply:
Ooos, yes, this should really be removed!